### PR TITLE
Fix golint failures of e2e/framework/[k-l]*.go

### PR DESCRIPTION
**Pull request by oomichi**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

This fixes golint failures on the following files:
- test/e2e/framework/kubelet_stats.go
- test/e2e/framework/log_size_monitoring.go

In addition, this replaces the combination of GetCPUSummary() and
FormatCPUSummary() with LogCPUSummary() in e2e/node/kubelet_perf.go
because that is completely same.

ref: https://github.com/kubernetes/kubernetes/issues/68026

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
thanks @oomichi 

/lgtm
/approve

File: test/e2e/node/kubelet_perf.go [hunk 1]
```diff
@@ -110,9 +110,9 @@ func runResourceTrackingTest(f *framework.Framework, podsPerNode int, nodeNames
 	framework.PrintPerfData(framework.ResourceUsageToPerfData(rm.GetMasterNodeLatest(usageSummary)))
 	verifyMemoryLimits(f.ClientSet, expectedMemory, usageSummary)
 
-	cpuSummary := rm.GetCPUSummary()
-	framework.Logf("%s", rm.FormatCPUSummary(cpuSummary))
+	rm.LogCPUSummary()
```
Comment (oomichi) [inline test/e2e/node/kubelet_perf.go]: Humm, this change makes GetCPUSummary() call twice.
So I feel now it is better to keep this itself without changes.

**Review by neolit123 — APPROVED** [review]
/lgtm
@oomichi feel free to `/hold cancel` if you think this is good.

**Pull request merged**